### ISSUE TITLE
Review JSDoc comments @fabianmoronzirfas

### ISSUE THREAD

**Issue by fabianmoronzirfas** [basiljs/basil.js#175]
- [ ] structure.js
- [ ] transformation.js
- [ ] typography.js

**Comment by fabianmoronzirfas** [comment]
basil.js/data.js at master · basiljs/basil.js
https://github.com/basiljs/basil.js/blob/master/src/includes/data.js#L232

Is this line needed? 

This more or less a reminder for myself. Is this jsDoc comment needed?

**Comment by b-g** [comment]
I don't think so ... most likely a hack to have the old docs look a certain way or a mistake.

**Comment by fabianmoronzirfas** [comment]
Oki. Thanks.
